lua/cmp/view.lua

```diff
@@ -239,7 +259,9 @@ view.on_entry_change = async.throttle(function(self)
       if not self:visible() then
         return
       end
-      self.docs_view:open(e, self:_get_entries_view():info())
+      if self.docs_view:visible() then
```

Comment:
Sorry. I didn't understand why this was needed. Could you explain it?

Reply:
This check is needed in order to correctly update the documentation window when the selected entry changes.

- If the documentation window is currently closed, then it won't be opened when the entry changes.
- If it's open, then we ensure that its content reflects the new entry.

Reply:
Hmm the issue with this is that the docs will initially be hidden. And so I think we do need the `docs_initially_visible`, or do you have a better idea of how to address this issue?

Reply:
@hrsh7th what are your thoughts on this :) ?